inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convert.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/convert.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+std::vector<std::vector<std::vector<size_t>>> inShapes = {
+        {{1}},
+        {{1, 2}},
+        {{2, 3, 4, 5}},
+        {{1, 2, 3, 4, 5}},
+};
+
+std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32,
+                                                         InferenceEngine::Precision::FP16};
+
+INSTANTIATE_TEST_CASE_P(NoReshape, ConvertLayerTest,
+                        ::testing::Combine(
+                                ::testing::ValuesIn(inShapes),
+                                ::testing::ValuesIn(netPrecisions),
+                                ::testing::ValuesIn(netPrecisions),
```

Comment:
I guess that pretty common scenario its conversion from floating to integer precisions or vice versa, so maybe to add cases with U8 I8 target precisions?

Reply:
Agree, TBD

Reply:
Done